### PR TITLE
[5264] - Bulk recommend CSV import

### PR DESCRIPTION
**Pull request by d-a-v-e**
### Context
MVP for CSV import (see trello ticket)
### Changes proposed in this pull request
...

### Guidance to review
...

### Important business
...

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [x] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?

### CODE REVIEW

**Comment by d-a-v-e** [comment]
[[L] Bulk QTS: CSV import](https://trello.com/c/GKKWlEva/5264-l-bulk-qts-csv-import)

File: app/models/bulk_update.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module BulkUpdate
```
[the remark this answers is not in the document]
Comment (edwardhorsford) [reply app/models/bulk_update.rb:1]: In the future we may have other bulk imports of things - is that relevant?

[the remark this answers is not in the document]
Comment (d-a-v-e) [reply app/models/bulk_update.rb:1]: yeah - we've made it a bit more generic to leave space for extending the use. the new tables are prefixed with bulk_update

**Review by markhallen — CHANGES_REQUESTED** [review]
Looks good so far!

File: db/migrate/20230216164121_create_bulk_update_recommendations_uploads.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateBulkUpdateRecommendationsUploads < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bulk_update_recommendations_uploads do |t|
+      t.timestamps
+
+      t.belongs_to :user, null: false, foreign_key: true
```
[the remark this answers is not in the document]
Comment (d-a-v-e) [reply db/migrate/20230216164121_create_bulk_update_recommendations_uploads.rb:8]: we'd end up with orphaned bulk uploads, is it better that - for now - an attempt at deleting a user with bulk uploads would throw an exception?

**Review by markhallen — APPROVED** [review]

**Pull request merged**